src/components/Editor/SearchBar.js

```diff
@@ -332,6 +322,29 @@ const SearchBar = React.createClass({
         buttonClass: "big"
       })
     );
+  },
+
+  renderBottomBar() {
+    if (!isEnabled("searchModifiers") || !isEnabled("functionSearch")) {
+      return;
+    }
+
+    return dom.div(
+      { className: "search-mod-bar" },
+      this.renderSearchModifiers()
+    );
+  },
+
+  render() {
+    if (!this.state.enabled) {
+      return dom.div();
+    }
+
+    return dom.div(
+      { className: "search-bar" },
+      this.renderSearchField(),
```

Comment:
I considered that but we have `this.searchInput` that I didn't want to confuse it with.